### PR TITLE
bugfix: don't check quorum for evaluate stage proposals

### PR DESCRIPTION
**Pull request by n13**
the quorum is dynamic and always changing depending on the number of proposals, number of eligible voters, and their vote power. Therefore we only check quorum on the initial vote. Once a proposal has been voted in and is in eval stage, we only check unity, in case someone changed their mind. But we don't check quorum.

@IanMendozaJaimes  please check

### CODE REVIEW

**Review by IanMendozaJaimes — APPROVED** [review]
It looks great!

File: src/seeds.proposals.cpp [hunk 1]
```diff
@@ -225,11 +225,20 @@ void proposals::onperiod() {
...
         double fav = double(pitr->favour);
         bool passed = pitr->favour > 0 && fav >= double(pitr->favour + pitr->against) * majority;
-        bool valid_quorum = utils::is_valid_quorum(voters_number, quorum, total_eligible_voters);
         name prop_type = get_type(pitr->fund);
         bool is_alliance_type = prop_type == alliance_type;
         bool is_campaign_type = prop_type == campaign_type;
 
+        bool valid_quorum = false;
+
+        if (pitr->status != status_evaluate) {
```
Comment (IanMendozaJaimes) [inline src/seeds.proposals.cpp:234]: Sorry, I got confused.

I think the if statement is doing the opposite you described, right now, the quorum is immediately valid when the proposal is not in evaluate state and when the proposal is in evaluate state it is checking if the quorum is valid. I think this `pitr->status != status_evaluate` should be `pitr->status == status_evaluate`

**Review by IanMendozaJaimes — COMMENTED** [review]
Sorry, I got confused at first, I think the if statement is doing the opposite you described.

File: src/seeds.proposals.cpp (hunk 1 above)
Comment (n13) [reply src/seeds.proposals.cpp:234]: Thanks, you're right - that's why I need reviews ;)

**Comment by n13** [comment]
Applied Ian's suggestions - deploying

**Pull request merged**